buildkite/build_project.sh

```diff
@@ -218,10 +218,14 @@ case "$REPO_FULL_NAME" in
         ;;
 
     ldc-developers/ldc)
-        git submodule update --init
+        git submodule update --init --depth 10 runtime/druntime runtime/phobos
```

Comment:
Are we sure that `--depth 10` is always going to work and not result in:

```
fatal: reference is not a tree: 41e9dece3fc6c2bc966cfa327ae8160f67a4c569
```

As with `--depth 1`?

Reply:
No, I've had it fail up to 15-25 too, don't know git well enough to say why.

Reply:
Ah then maybe just remove `--depth` for now and we can figure this out in a different PR?

Reply:
See also: https://stackoverflow.com/questions/2144406/how-to-make-shallow-git-submodules

Reply:
Installed git versions on the clients are 2.7.4 and 2.11.0